packages/mdctl-import-adapter/index.js

```diff
@@ -102,8 +102,11 @@ class ImportFileTreeAdapter extends EventEmitter {
 
   async loadPackageFromObject() {
     const { packageData, input, format } = privatesAccessor(this),
-          section = new ImportSection(packageData, 'package', `package.${format}`, input)
-    return { results: [section.content], blobResults: [] }
+          { content } = new ImportSection(packageData, 'package', `package.${format}`, input)
```

Comment:
Note that the only effect of the `new ImportSection` call here is to map `packageData` to the new object's `content` property, so lines 104-105 could be replaced with just:
```js
const { packageData: content, input, format } = privatesAccessor(this)
```